pkg/operator/ceph/object/user/controller.go

```diff
@@ -309,44 +308,11 @@ func (r *ReconcileObjectStoreUser) initializeObjectStoreContext(u *cephv1.CephOb
 	if err != nil {
 		return errors.Wrapf(err, "Multisite failed to set on object context for object store user")
 	}
-
-	r.objContext = objContext
-	if store.Status == nil {
-		return errors.New("failed to initialize ceph object user because unknown object store status")
-	}
-
-	if _, ok := store.Status.Info["endpoint"]; ok {
-		r.objContext.Endpoint = store.Status.Info["endpoint"]
-	} else {
-		return errors.New("endpoint is missing in the status Info")
-	}
-
-	if _, ok := store.Status.Info["secureEndpoint"]; ok {
-		r.objContext.Endpoint = store.Status.Info["secureEndpoint"]
-	}
```

Comment:
Now that the `NewMultisiteAdminOpsContext` generates endpoint info for the referenced CephObjectStore, there is no need to wait for the status to be ready to get the endpoint information.